catalogue/webapp/pages/concept.tsx

```diff
@@ -128,81 +108,75 @@ export const ConceptPage: NextPage<Props> = ({
   const hasImagesTabs = !!(imagesBy?.totalResults && imagesAbout?.totalResults);
 
   return (
-    // TODO fill meta information; who decides this?
+    // TODO fill meta information; wait for confirmation on description
     <CataloguePageLayout
       title={conceptResponse.label}
-      description="<TBC>"
+      description={`Find out more about ${conceptResponse.label} by browsing related works and images from the Wellcome Collection.`}
       url={{ pathname: `/concepts/${conceptResponse.id}`, query: {} }}
       openGraphType="website"
       siteSection="collections"
       jsonLd={{ '@type': 'WebPage' }}
       hideNewsletterPromo={true}
     >
-      <ConceptHero
-        v={{ size: 'xl', properties: ['padding-top', 'padding-bottom'] }}
-      >
+      <ConceptHero>
         <div className="container">
-          <ConceptDescription>
-            <h1 className="font-intb">{conceptResponse.label}</h1>
-            {/* TODO dynamise */}
-            {FAKE_DATA.description && (
-              <p className="font-size-5">{FAKE_DATA.description}</p>
-            )}
-            {/* TODO dynamise */}
-            {FAKE_DATA.urls?.length > 0 &&
-              FAKE_DATA.urls.map(link => {
-                /* TODO Could they be internal links? Check if external to display arrow, decide on rel. */
-                return (
-                  <a
-                    key={link.url}
-                    href={link.url}
-                    rel="nofollow"
-                    className={font('intr', 6)}
-                  >
-                    {link.label} ↗
-                  </a>
-                );
-              })}
-          </ConceptDescription>
+          <TypeLabel>{conceptResponse.type}</TypeLabel>
+          <Space v={{ size: 's', properties: ['margin-top', 'margin-bottom'] }}>
+            {/* TODO it all comes out as subject, chase if it doesn't change */}
```

Comment:
I believe that's what Jonathan wants to touch on today, but I didn't get the details. It technically won't change anything for me though, so that Todo is just a reminder to myself to check in on that conversation.